### PR TITLE
#10168 - Fix to Can't open settings if browser zoom level > 100% part 2

### LINKED ISSUE #10168: Can't open settings if browser zoom level > 100%
**Describe the bug**
If zoom level in the browser is set to more than 100% there is no way to open settings in the menu. On 100% it's fine.
The menu is not scrollable.

**Steps to reproduce (REQUIRED)**
- set zoom level in browser settings to 125%
- open metamask menu - you won't see setting in the bottom.

**Screenshots**
Screen: http://joxi.net/l2ZM6p3i7Zn33m

**Browser details (please complete the following information):**
 - OS: MacOS
 - Browser: Tested in Chrome, Edge.
 - MetaMask Version: 8.1.11

### PR DESCRIPTION
**Pull request by digiwand**
**Fixes:** #10168

we can merge this PR following this Revert PR merge: https://github.com/MetaMask/metamask-extension/pull/13459

**Explanation:** 
When users are viewing the extension with > 100% zoom, they are unable to access the "Settings" option in the account menu. 
 
Changing the position from `fixed` to `absolute` on the `.account-menu` enables the user to scroll to the "Settings" option. This comes with a slight UI/UX change. When the user has not scrolled, the position is preserved because the parent element is `.app`. When the user scrolls, the positioning remains in place at the top instead of relative to the viewport.

**Manual testing steps:**  
  - On Chrome or Edge, set the browser settings to 125%
    - For Chrome:
      - Go to Settings (`Cmd` + `,`)
      - Click "Appearance" from left navigation
      -  Update "Page zoom" setting to "125%)
  - Open MetaMask
  - Open Account Menu
  - Scroll on menu to get to Settings

**Bug:**

**Previous Main Content Scroll Behavior:**

**New Main Content Scroll Behavior:** 

**Verified in Extension Popup:**

### CODE REVIEW

**Comment by DELJUJAZI** [comment]
0 ingreso

**Review by DELJUJAZI — DISMISSED** [review]

**Review by Gtonizuka — DISMISSED** [review]

**Comment by jpuri** [comment]
@digiwand : looking at screenshot of fix above I am wondering it we can have just the account popup scrolling rather than to have to scroll whole page.

**Review by brad-decker — DISMISSED** [review]

**Comment by georgewrmarshall** [comment]
@jpuri @digiwand IMO this should have been absolute to begin with? The menu should be absolutely positioned depending on where the menu anchor is, or in this case the account identicon. I think this is the right solution for now

File: ui/components/app/account-menu/index.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
   color: var(--white);
 
   @media screen and (max-width: $break-small) {
-    right: calc(((100vw - 100%) / 2) + 8px);
+    right: 1rem;
```
Comment (georgewrmarshall) [inline ui/components/app/account-menu/index.scss:13]: I think we should move towards a convention of using `px` values for bounding box attributes e.g. `padding` and `margin`. `rem`s/`em`s don't provide any accessibility benefit for things other than fonts. Pixels are also more aligned with how designers work. Have some notes on CSS guidelines but have yet to publish them anywhere will add them to the current documentation effort. Also open to discussions on this!

```suggestion
    right: 16px;
```

**Review by georgewrmarshall — COMMENTED** [review]
Agree with the solution here one thing about units used but otherwise LGTM!

**Comment by digiwand** [comment]
nooooo, sorry, I had to dismiss the reviews. One line change here: d0562bc 
@brad-decker @Gtonizuka @DELJUJAZI @jpuri

**Comment by digiwand** [comment]
@georgewrmarshall I thought the fixed behavior was funny 😅

Thanks for the confirmation and the helpful SCSS comment!

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Review by danjm — APPROVED** [review]

**Pull request merged**